// ERROR:
fn main() {
  let y = 3;
}